Makefile

```diff
@@ -177,10 +187,11 @@ help:  # Display this help
 ##@ generate:
 
 ALL_GENERATE_MODULES = core kubeadm-bootstrap kubeadm-control-plane
+ALL_GENERATE_OPENAPI = api/v1beta1 $(EXP_DIR)/runtime/hooks/api/v1alpha1
```

Comment:
I think I would move this to the target where it's used (probably just inline it) as we usually have the packages in the targets. (it's also only used there)